app/database/seed.server.ts

```diff
@@ -90,9 +90,11 @@ const supabaseAdmin = createClient(SUPABASE_URL, SUPABASE_SERVICE_ROLE, {
 
 const prisma = new PrismaClient();
 
-const email = "hello@supabase.com";
+const email = process.env.ADMIN_EMAIL || "hello@supabase.com";
```

Comment:
Something else I noticed here. This works but its not fitting with the way we usually do it in shelf. Please take a look at the file `env.ts` and how we manage and export/import env variables.

Reply:
Will work on it in few hours

Reply:
@DonKoko I think the new Changes should follow the practice.

Reply:
Thanks @jugrajsingh. Thats perfect. I am merging the PR.